fire/SFParamsMod.F90

```diff
@@ -7,68 +7,58 @@ module SFParamsMod
-   ! seen outside of this module.
 
    !
    ! this is what the user can use for the actual values
    !
-   real(r8),protected :: SF_val_fdi_a
-   real(r8),protected :: SF_val_fdi_b
-   real(r8),protected :: SF_val_fdi_alpha
-   real(r8),protected :: SF_val_miner_total
-   real(r8),protected :: SF_val_fuel_energy
-   real(r8),protected :: SF_val_part_dens
-   real(r8),protected :: SF_val_miner_damp
-   real(r8),protected :: SF_val_max_durat
-   real(r8),protected :: SF_val_durat_slope
-   real(r8),protected :: SF_val_drying_ratio
-   real(r8),protected :: SF_val_CWD_frac(NCWD)
-   real(r8),protected :: SF_val_max_decomp(NFSC)
-   real(r8),protected :: SF_val_SAV(NFSC)
-   real(r8),protected :: SF_val_FBD(NFSC)
-   real(r8),protected :: SF_val_min_moisture(NFSC)
-   real(r8),protected :: SF_val_mid_moisture(NFSC)
-   real(r8),protected :: SF_val_low_moisture_Coeff(NFSC)
-   real(r8),protected :: SF_val_low_moisture_Slope(NFSC)
-   real(r8),protected :: SF_val_mid_moisture_Coeff(NFSC)
-   real(r8),protected :: SF_val_mid_moisture_Slope(NFSC)
-
-   character(len=param_string_length),parameter :: SF_name_fdi_a = "fates_fdi_a"
-   character(len=param_string_length),parameter :: SF_name_fdi_b = "fates_fdi_b"
-   character(len=param_string_length),parameter :: SF_name_fdi_alpha = "fates_fdi_alpha"
-   character(len=param_string_length),parameter :: SF_name_miner_total = "fates_miner_total"
-   character(len=param_string_length),parameter :: SF_name_fuel_energy = "fates_fuel_energy"
-   character(len=param_string_length),parameter :: SF_name_part_dens = "fates_part_dens"
-   character(len=param_string_length),parameter :: SF_name_miner_damp = "fates_miner_damp"
-   character(len=param_string_length),parameter :: SF_name_max_durat = "fates_max_durat"
-   character(len=param_string_length),parameter :: SF_name_durat_slope = "fates_durat_slope"
-   character(len=param_string_length),parameter :: SF_name_drying_ratio = "fates_drying_ratio"
-   character(len=param_string_length),parameter :: SF_name_CWD_frac = "fates_CWD_frac"
-   character(len=param_string_length),parameter :: SF_name_max_decomp = "fates_max_decomp"
-   character(len=param_string_length),parameter :: SF_name_SAV = "fates_SAV"
-   character(len=param_string_length),parameter :: SF_name_FBD = "fates_FBD"
-   character(len=param_string_length),parameter :: SF_name_min_moisture = "fates_min_moisture"
-   character(len=param_string_length),parameter :: SF_name_mid_moisture = "fates_mid_moisture"
-   character(len=param_string_length),parameter :: SF_name_low_moisture_Coeff = "fates_low_moisture_Coeff"
-   character(len=param_string_length),parameter :: SF_name_low_moisture_Slope = "fates_low_moisture_Slope"
-   character(len=param_string_length),parameter :: SF_name_mid_moisture_Coeff = "fates_mid_moisture_Coeff"
-   character(len=param_string_length),parameter :: SF_name_mid_moisture_Slope = "fates_mid_moisture_Slope"
-
+   real(r8),public,protected :: SF_val_fdi_a
```

Comment:
@glemieux , if a variable is defined as protected, does it make it implicitly public?  I poked around on the internet and could not find an answer.

Reply:
That's a good question.  Chivers' book isn't explicit but the example code they give to show suggests that it's not the case that it's implicitly public.  I'll write up a quick script and test it out in a bit.

Reply:
@rgknox based on the [small sample code](https://github.com/glemieux/fates-scratch/blob/master/fortran-scratch/ProtectedTestMain.F90) I worked up it looks like setting something as `protected` does not make it implicitly `public`.